cloud/disk_manager/deploy/helm/disk-manager/values.schema.json

```diff
@@ -0,0 +1,210 @@
+        }
+      }
+    },
+    "image": {
+      "type": "object",
+      "form": true,
+      "properties": {
+        "repository": {
+          "type": "string",
+          "title": "Path to Disk-Manager docker image in the repository"
+        },
+        "pullPolicy": {
+          "type": "string",
+          "enum": ["IfNotPresent", "Always"],
+          "title": "K8s image pull policy for disk manager image"
+        },
+        "tag": {
+          "type": "string",
+          "title": "Disk-Manager docker image tag to deploy"
+        }
+      }
+    },
+    "tasksConfig": {
+      "type": "object",
+      "properties": {
+        "runnersCount": {
+          "type": "integer",
+          "title": "Number of runners of tasks processing"
+        },
+        "stalkingRunnersCount": {
+          "type": "integer",
+          "title": "Number of stalking runners of tasks processing"
+        }
+      }
+    },
+    "zones": {
+      "patternProperties": {
+        ".*": {
+          "type": "object",
+          "properties": {
+            "nbsSvms": {
+              "type": "array",
+              "title": "List of NBS server hosts",
+              "items": {
+                "type": "string"
+              }
+            },
+            "nbsPort": {
+              "type": "integer",
+              "title": "Port of NBS server (NBS servers are expected to run on the same port)"
+            }
+          }
+        }
+      }
+    },
+    "nbs": {
+      "type": "object",
+      "properties": {
+        "nbsControl": {
+          "type": "object",
+          "properties": {
+            "Insecure": {
+              "type": "boolean",
+              "title": "Use insecure NBS server GRPC API"
+            },
+            "EnableThrottlingForMediaKinds": {
+              "type": "array",
+              "title": "List of media kinds to enable throttling for",
+              "items": {
+                "type": "string"
+              }
+            },
+            "UseGZIPCompression": {
+              "type": "boolean",
+              "title": "Use GZIP compression in NBS"
+            }
+          }
+        }
+      }
+    },
+    "nfs": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "title": "NFS file storage enabled"
+        },
+        "insecure": {
+          "type": "boolean",
+          "title": "Do not use TLS for connecting to the NFS API"
+        },
+        "disableAuthentication": {
+          "type": "boolean",
+          "title": "Disable authentication for NFS file share"
+        }
+      }
+    },
+    "ydb": {
+      "type": "object",
+      "properties": {
+        "Secure": {
+          "type": "boolean"
+        },
+        "Endpoint": {
+          "type": "string"
+        },
+        "Port": {
+          "type": "integer"
+        },
+        "Database": {
+          "type": "string"
+        },
+        "RootPath": {
```

Comment:
property names are inconsistent 
most start with `lowerCase` but some start with `UpperCase` let's follow one convention for all properties for example `lowerCase`